### PR TITLE
BAEL-1045 Lambda Behave

### PR DESCRIPTION
**Pull request by mansi2392**

### CODE REVIEW

File: testing/src/main/java/com/baeldung/lambdabehave/Calculator.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        this.x = x;
+        this.y = y;
+    }
+
+    public int add() {
+        return this.x + this.y;
+    }
+
+    public int divide(int a, int b) {
+        if (b == 0) {
```
Comment (pivovarit) [inline testing/src/main/java/com/baeldung/lambdabehave/Calculator.java]: No need for that `if` block. ArithmeticException is throw automatically when the division by zero happens

File: testing/src/test/java/com/baeldung/lambdabehave/CalculatorTest.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            it.should("Add the given numbers", expect -> {
+                expect.that(calculator.add()).is(3);
+            });
+            it.should("Throw an exception if divide by 0", expect -> {
+                expect.exception(ArithmeticException.class, () -> {
+                    calculator.divide(1, 0);
+                });
+            });
+            it.uses(2, 3, 5)
+                    .and(23, 10, 33)
```
Comment (pivovarit) [inline testing/src/test/java/com/baeldung/lambdabehave/CalculatorTest.java]: Please reformat files using our classic formatter. The continuation indentation should be equal to 2 spaces.

File: testing/src/main/java/com/baeldung/lambdabehave/Calculator.java (hunk 1 above)
Comment (mansi2392) [inline testing/src/main/java/com/baeldung/lambdabehave/Calculator.java]: Right, thanks @pivovarit . Done.

File: testing/src/test/java/com/baeldung/lambdabehave/CalculatorTest.java (hunk 2 above)
Comment (mansi2392) [inline testing/src/test/java/com/baeldung/lambdabehave/CalculatorTest.java]: Done

**Pull request merged**